src/hierarchies/mfa_hierarchy.cc

```diff
@@ -0,0 +1,248 @@
+#include "mfa_hierarchy.h"
+
+#include <google/protobuf/stubs/casts.h>
+
+#include <Eigen/Dense>
+#include <stan/math/prim/prob.hpp>
+#include <vector>
+
+#include "algorithm_state.pb.h"
+#include "hierarchy_prior.pb.h"
+#include "ls_state.pb.h"
+#include "src/utils/rng.h"
+
+double MFAHierarchy::like_lpdf(const Eigen::RowVectorXd & datum) const {
+
+    return stan::math::normal_lpdf(
+        datum, state.mu + state.Lambda*state.lambda.transpose(), state.psi.cwiseSqrt());
+}
+
```

Comment:
Can we assume that the prior behaves exactly as specified in the IMIFA paper or do we need to keep it more general? For example, Lambda is N(0,I) so we just sampled from univariate normal distributions indipendently.

Reply:
Yes I think it's fine! At least for the moment